scaffolds/java/scripts/test

```diff
@@ -2,6 +2,6 @@
 
 cd $(dirname $0)/..
 
-./gradle check
+./gradle test
```

Comment:
What's the purpose of this change ?

Reply:
To my knowledge, `check` is not commonly used, while `test` is widely used.
Furthermore, `check` makes sense when adding some extra task to the verification process, which is not the case of this scaffold.
[source](https://stackoverflow.com/a/50105980/3790512)

Reply:
ok !